docs/apis/plugintypes/tiny/index.md

```diff
@@ -0,0 +1,586 @@
+---
+title: TinyMCE Editor Plugins
+tags:
+  - editor_tiny
+  - tiny_
+  - Editor
+  - Text editor
+  - HTML editor
+  - WYSIWYG
+  - TinyMCE
+---
+
+Moodle includes the TinyMCE text editor as standard from Moodle 4.1, and it can be installed from the plugins database for Moodle versions 3.11, and 4.0.
```

Comment:
I guess for now it hasn't been uploaded to the plugins database yet, right? (at least, I can't find it)

Reply:
Correct - I don't plan to upload it until about a week after it's been integrated to core.